examples/reminders/lib/base/data_sources/local/reminders_local_data_source.dart

```diff
@@ -57,6 +58,7 @@ class RemindersLocalDataSource {
       id: id,
       title: title,
       complete: complete,
+      completeUpdated: completeUpdated,
```

Comment:
As we discussed  the rx_bloc_list now supports the needed infrastructure for checking old/new objects so please adapt this functionality accordingly